fieldservice_equipment_type/models/fsm_equipment_type.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FSMEquipmentType(models.Model):
+    _name = "fsm.equipment.type"
+    _description = "FSM Equipment Type"
+
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
```

Comment:
Why this field has to be required ?

Reply:
@BojanOD Could you help with this question?